tests/api.c

```diff
@@ -54815,6 +54840,192 @@ static int test_wolfSSL_dtls_set_mtu(void)
     return 0;
 }
 
+#if defined(HAVE_IO_TESTS_DEPENDENCIES) && !defined(SINGLE_THREADED) && \
+    defined(WOLFSSL_DTLS)
+
+static void test_wolfSSL_dtls12_fragments_spammer(WOLFSSL* ssl)
```

Comment:
NIT: it would be nice to have some of these hardcoded number as defines, maybe with a very specific prefix (WOLFSSL_TEST_?)

Reply:
I'm not sure if it makes sense since this will be used only here. Maybe if we start making more generic functions to generate dummy packets then it will make sense.